components/molecules/Greeting.js

```diff
@@ -3,9 +3,11 @@ import { Message } from '@dts-stn/decd-design-system'
 export default function Greeting(props) {
   return (
     <div>
-      <div className="grid grid-cols-1 divide-y divide-red-600 font-display">
-        <div className="py-4 text-4xl font-bold">{`${props.greeting} ${props.name}`}</div>
-        <div className="py-4 text-xl">{props.welcome}</div>
+      <div className="grid grWid-cols-1 divide-y divide-red-600 font-display">
+        <h1 className="py-4 text-3xl font-bold text-gray-darker">
+          {props.myBenefitsAndServices}
+        </h1>
+        <h2 className="py-4 text-3xl font-bold">{`${props.greeting} ${props.name}`}</h2>
```

Comment:
We actually cant put text like this in a header, as a header needs to describe the content in it's section for accessibility.